src/mpyl/steps/build/docker_build.py

```diff
@@ -80,6 +80,14 @@ def execute(self, step_input: Input) -> Output:
                 arg.key: arg.get_value(step_input.run_properties.target)
                 for arg in step_input.project.build.args.plain
             }
+            | {
+                k: v
+                for (k, v) in {
+                    arg.key: os.getenv(arg.secret_id)
```

Comment:
shouldn't we do some sort of check that it can find the secret in the env vars?

Reply:
Yeah maybe that's better, I'll just throw if an env var is not set